### PR TITLE
Core/Arena: Add config option for Arena.QueueAnnoncer

### LINKED ISSUE #304: Arena.QueueAnnouncer does not work
**Description**:

**Current behaviour**: Tell us what happens

Sand announcer does not work when activated
worldserver.conf and worldserver.conf.dist
Arena.QueueAnnouncer.Enable = 1

**Expected behaviour**: Tell us what should happen instead

should announce the sand equipments

**Branch(es)**: 0.x / 1.x / master (Specify the branch(es) affected by this issue) 

**AC hash/commit**:  
https://github.com/azerothcore/azerothcore-wotlk/commit/1eb07581fed648a5791ca48fbb300f35798633de

**Other Customizations**: Are you use core/db patches or other customizations? If yes please specify them here.

NO

### PR DESCRIPTION
**Pull request by Blumfield**
**Changes proposed:**

-  Add the needed code for the config option

**Target branch(es):** 1.x/2.x etc. Master

**Issues addressed:** Closes #304

**Tests performed:** (Does it build, tested in-game, etc) Yes, tested ingame see screenshot
http://i.imgur.com/7Og58Ao.jpg

**Known issues and TODO list:** None

- [ ] 
- [ ]

### CODE REVIEW

**Comment by Sworken** [comment]
Great work thank you

File: src/game/Battlegrounds/BattlegroundQueue.cpp [hunk 1]
```diff
@@ -145,6 +145,14 @@ GroupQueueInfo* BattlegroundQueue::AddGroup(Player* leader, Group* grp, PvPDiffi
     ginfo->_bracketId = bracketId;
     ginfo->_groupType = index;
 
+    //announce world (this don't need mutex)
```
Comment (ShinDarth) [inline src/game/Battlegrounds/BattlegroundQueue.cpp]: `` // announce world (this doesn't need mutex)``

**Comment by ShinDarth** [comment]
@Sworken did you test it?

File: src/game/Battlegrounds/BattlegroundQueue.cpp (hunk 1 above)
Comment (Blumfield) [inline src/game/Battlegrounds/BattlegroundQueue.cpp]: do you want me to add a space ?

Comment (ShinDarth) [inline src/game/Battlegrounds/BattlegroundQueue.cpp]: yes and change ``don't`` with ``doesn't`` please

**Comment by Sworken** [comment]
@ShinDarth  this patch works!

**Comment by ShinDarth** [comment]
thanks @Blumfield  for the PR and @Sworken for feedback, merged

**Pull request merged**